rustler/src/wrapper/nif_interface.rs

```diff
@@ -337,16 +337,16 @@ pub unsafe fn enif_consume_timeslice(env: NIF_ENV, percent: c_int) -> c_int {
     erlang_nif_sys::enif_consume_timeslice(env, percent)
 }
 
-pub unsafe fn enif_schedule_nif(
-    env: NIF_ENV,
-    fun_name: *const c_uchar,
-    flags: c_int,
-    dtor: Option<unsafe extern "C" fn(env: NIF_ENV, argc: c_int, argv: *const NIF_TERM)>,
-    argc: c_int,
-    argv: *const NIF_TERM,
-) -> NIF_TERM {
-    erlang_nif_sys::enif_schedule_nif(env, fun_name, flags, dtor, argc, argv)
-}
+//pub unsafe fn enif_schedule_nif(
```

Comment:
Did you mean to comment this out?

Reply:
Yep, the function signature is incorrect in the erlang_nif-sys crate. I fixed the type signature here and submitted a PR to fix it upstream. I commented it out while waiting so that it would still compile.

Planning to uncomment before merging